addons/hr_holidays/hr_holidays_view.xml

```diff
@@ -587,19 +587,22 @@
                     </group>
                 </xpath>
                 <xpath expr="//div[@name='button_box']" position="inside">
+                    <field name="show_leaves" invisible="1"/>
                     <button name="%(act_hr_employee_holiday_request_approved)d"
                             icon="fa-calendar"
                             class="oe_stat_button"
                             type="action"
-                            groups="base.group_hr_user">
-                        <field name="approved_leaves_count" widget="statinfo"/>
+                            attrs="{'invisible': [('show_leaves','=', False)]}"
+                            groups="base.group_user" help="Leaves to Process">
+                        <field string="Leaves to Pro." name="approved_leaves_count" widget="statinfo"/>
```

Comment:
string is larger then button fix width (in oe_state button). So I use this way reference from following.


I is this not okay then can I write full string, but string overflow going hidden ? Thanks
![download](https://cloud.githubusercontent.com/assets/7599774/6634673/b3f221de-c981-11e4-9836-2b9f8122905e.png)